lib/karto_sdk/include/karto_sdk/Karto.h

```diff
@@ -5658,7 +5658,10 @@ class LocalizedRangeScan : public LaserRangeScan
       kt_int32u beamNum = 0;
       for (kt_int32u i = 0; i < pLaserRangeFinder->GetNumberOfRangeReadings(); i++, beamNum++) {
         kt_double rangeReading = GetRangeReadings()[i];
-        if (!math::InRange(rangeReading, pLaserRangeFinder->GetMinimumRange(), rangeThreshold)) {
+        if (rangeReading < pLaserRangeFinder->GetMinimumRange()) {
```

Comment:
Why not using that `InRange` line that was there before `math::InRange(rangeReading, pLaserRangeFinder->GetMinimumRange(), rangeThreshold)`? This seems to remove the far-check